tutorials/finetune_lora.md

```diff
@@ -59,25 +59,21 @@ python finetune/lora.py --quantize "bnb.nf4-dq"
 
 The table below lists a comparison with different settings on a StableLM 3B model finetuned with LoRA on Alpaca for 5,000 iterations using a microbatch size of 4:
 
-| Settings                                                | Training Memory  | Training Time  | Loss      | Inference Memory |
-|---------------------------------------------------------|------------------|----------------|-----------|------------------|
-| Default (bfloat16-mixed)                                | 33.50 GB         | 591.78s        | 0.9207    | 7.61 GB          |
-| --precision "bf16-true"                                 | 15.86 GB         | 592.14s        | 0.9180    | 7.61 GB          |
-| --quantize "bnb.nf4"                                    | 22.34 GB         | 944.93s        | 0.9417    | 3.25 GB          |
-| --quantize "bnb.nf4-dq"                                 | 22.18 GB         | 962.23s        | 0.9383    | 3.08 GB          |
-| --precision "bf16-true"  --quantize "bnb.nf4"           | 14.81 GB         | 802.02s        | 0.9408    | 3.25 GB          |
-| --precision "bf16-true"  --quantize "bnb.nf4-dq"        | 14.65 GB         | 802.94s        | 0.9384    | 3.08 GB          |
+| Settings                                    | Training Memory | Training Time | Loss   | Inference Memory |
+|---------------------------------------------|-----------------|---------------|--------|------------------|
+| Default (bf16-mixed)                        | 38.01 GB        | 591.78s       | 0.9207 | 21.49 GB         |
+| --precision bf16-true                       | 19.40 GB        | 592.14s       | 0.9180 | 7.35 GB          |
+| --precision bf16-true --quantize bnb.nf4    | 19.13 GB        | 802.02s       | 0.9408 | 3.26 GB          |
+| --precision bf16-true --quantize bnb.nf4-dq | 18.97 GB        | 802.94s       | 0.9384 | 3.09 GB          |
 
 The advantages of QLoRA-style quantization are more pronounced in larger models, such as Llama 2 7B. The table below summarizes the results for Llama 2 7B on Alpaca for 5,000 iterations using a microbatch size of 4:
 
-| Settings                                            | Training Memory  | Training Time  | Loss   | Inference Memory |
```

Comment:
@rasbt I'm unable to reproduce your original training memory results after checking out https://github.com/Lightning-AI/lit-gpt/pull/275. Can you help?